tests/mail/tests.py

```diff
@@ -1478,3 +1478,23 @@ def test_server_stopped(self):
         backend.open()
         self.server.stop()
         backend.close()
+
+
+class SMTPBackendStoppedServerFailSilentlyTest(SMTPBackendTestsBase):
+    """
+    This test requires a separate class, because it shuts down the
+    FakeSMTPServer started in setUpClass(). It cannot be restarted
+    ("RuntimeError: threads can only be started once").
+    """
+
+    def test_fail_silently_on_connection_error(self):
+        """
+        Using fail_silently with the SMTPBackend doesn't raise an error
+        when the server is stopped.
+        """
+        self.server.stop()
+        backend = smtp.EmailBackend(username='', password='')
+        with self.assertRaises(OSError):
```

Comment:
Looks like `socket.error` works on Python 2 and 3. Is there a difference on Windows?